packaging/package.sh

```diff
@@ -3,11 +3,23 @@ set -euxo pipefail
 
 BASEDIR=$(dirname "$0")
 OUTPUT=$1
-GOARCH=$(go env GOARCH)
 CODESIGN_IDENTITY=${CODESIGN_IDENTITY:-mock}
 PRODUCTSIGN_IDENTITY=${PRODUCTSIGN_IDENTITY:-mock}
 NO_CODESIGN=${NO_CODESIGN:-0}
 
+case "$(uname -m)" in
+    "x86_64")
+         GOARCH="amd64"
+        ;;
```

Comment:
Indentation might be a little bit off here?

Reply:
yeah, will fix this and merge